rpc/src/v1/tests/mocked/eth.rs

```diff
@@ -852,3 +853,28 @@ fn should_not_return_block_number() {
 
 	assert_eq!(eth_tester.io.handle_request(request), Some(response.to_owned()));
 }
+
+#[test]
+fn rpc_get_work_should_timeout() {
+	let eth_tester = EthTester::default();
+	eth_tester.miner.set_author(Address::from_str("d46e8dd67c5d32be8058bb8eb970870f07244567").unwrap());
+	eth_tester.client.set_latest_block_timestamp(get_time().sec as u64 - 1000);  // Set latest block to 1000 seconds ago
+	let hash = eth_tester.miner.map_sealing_work(&*eth_tester.client, |b| b.hash()).unwrap();
+
+	// Request with timeout of 0 seconds. This should work since we're disabling timeout.
+	let request = r#"{"jsonrpc": "2.0", "method": "eth_getWork", "params": [], "id": 1}"#;
```

Comment:
This test case shows that the API is backward compatible. I should probably also add a test case which explicitly passes in 0.